glossary.md

```diff
@@ -160,6 +280,10 @@ Schedulers are components in PD that generate scheduling tasks. Each scheduler i
 - `hot-region-scheduler`: Balances the distribution of hot Regions
 - `evict-leader-{store-id}`: Evicts all leaders of a node (often used for rolling upgrades)
 
+### SST
+
+Static Sorted Table, Sorted String Table, or Sorted Sequence Table (SST) is a file storage format used in RocksDB.
+
 ### Store
 
 A store refers to the storage node in the TiKV cluster (an instance of `tikv-server`). Each store has a corresponding TiKV instance.
```

Comment:
As we adding the PD component to this glossary should we also add the TiDB Server, TiFlash Server, and TiKV Server components as well for completeness?

Suggested additions:

### TiDB Server

The [TiDB server](https://docs.pingcap.com/tidb/stable/tidb-computing) is a stateless SQL layer that exposes the connection endpoint of the MySQL protocol to the outside. The TiDB server receives SQL requests, performs SQL parsing and optimization, and ultimately generates a distributed execution plan.

### TiFlash Server

The [TiFlash server](https://docs.pingcap.com/tidb/stable/tiflash-overview) is a special type of storage server. Unlike ordinary TiKV nodes, TiFlash stores data by column, mainly designed to accelerate analytical processing.

### TiKV Server

The [TiKV server](https://docs.pingcap.com/tidb/stable/tidb-storage) is responsible for storing data. TiKV is a distributed transactional key-value storage engine.